go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/cloudflare/cfssl v1.6.5
 	github.com/go-logr/logr v1.4.2
 	github.com/google/uuid v1.6.0
-	github.com/kong/kubernetes-configuration v0.0.0-20240801170722-d6586edd1b81
+	github.com/kong/kubernetes-configuration v0.0.0-20240802130201-3233f9f923a7
```

Comment:
Could we start tagging `github.com/kong/kubernetes-configuration` so we do not use main refs here?

Reply:
👍 8b1abef6aaf4d942a878996145442fa3d57f9d15